packages/release/release.spec

```diff
@@ -129,6 +129,7 @@ Requires: %{_cross_os}os
 Requires: %{_cross_os}pciutils
 Requires: %{_cross_os}policycoreutils
 Requires: %{_cross_os}procps
+Requires: (%{_cross_os}rdma-core if %{_cross_os}variant-platform(aws))
```

Comment:
nit: for consistency, you could move the conditionals to the end

Reply:
I have considered that too. But I added rdma-core package here to follow lexical ordering.